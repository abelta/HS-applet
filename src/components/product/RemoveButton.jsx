import React from 'react';
import PropTypes from 'prop-types';

const RemoveButton = ({ onClick }) => <button onClick={onClick}>Remove</button>;

RemoveButton.propTypes = {
  onClick: PropTypes.func.isRequired,
};

export default RemoveButton;
