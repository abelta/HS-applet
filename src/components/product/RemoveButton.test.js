import React from 'react';
import renderer from 'react-test-renderer';
import RemoveButton from './RemoveButton';

describe('RemoveButton', () => {
  const tree = renderer.create(<RemoveButton onClick={jest.fn} />).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
