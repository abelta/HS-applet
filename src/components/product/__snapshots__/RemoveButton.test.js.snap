// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoveButton renders correctly 1`] = `
<button
  className="btn-small waves-effect waves-light red white-text"
  onClick={[Function]}
>
  Remove
</button>
`;
